src/patternfly/components/List/list.scss

```diff
@@ -1,54 +1,48 @@
 @import "../../patternfly-utilities";
 
 .pf-c-list {
-  // Inline modifier variables
-  --pf-c-list--m-inline--MarginRight: var(--pf-global--spacer--md);
-
-  // Grid modifier variables
-  --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
-
-  // Content variables (no modifier)
-  --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
-  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
-  --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-list--MarginY: var(--pf-global--spacer--md);
+  --pf-c-list--MarginLeft: var(--pf-global--spacer--lg);
+  --pf-c-list--ul--nested--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
+  --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
+
+  padding-left: var(--pf-c-list--PaddingLeft);
+  margin: var(--pf-c-list--MarginY) 0 0 var(--pf-c-list--MarginLeft);
+
+  &:not(:last-child) {
+    margin-bottom: var(--pf-c-list--MarginY);
```

Comment:
@mcoker seeing as this list isn't in the context of the content component, does it need a bottom margin?

Reply:
Correct. The use of a list in the content component would create space around it since that's designed to flow with other content. This is just to create a list somewhere. It's similar to how an `h1` behaves in the content component compared to the `.pf-c-title` component.